test/integration/serverless_perf/write_message.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+filename="${HOME}/comment.md"
+echo "filename=$filename" >> "$GITHUB_OUTPUT"
+
+if [ "${VAR_COLD_START}" != 0 ]; then
+  echo -n ":warning::rotating_light: Warning, " > "$filename"
+else
+  echo -n ":inbox_tray: :loudspeaker: Info, " > "$filename"
+fi
+
+cat >> "$filename" << EOL
+this pull request increases the binary size of serverless extension by ${VAR_DIFF} bytes. Each MB of binary size increase means about 10ms of additional cold start time, so this pull request would increase cold start time by ${VAR_COLD_START}ms.
+
+<details>
+<summary>Debug info</summary>
+
+If you have questions, we are happy to help, come visit us in the [#serverless](https://dd.slack.com/archives/CBWDFKWV8) slack channel and provide a link to this comment.
+
+These dependencies were added to the serverless extension by this pull request:
+
+\`\`\`
+${VAR_DEPS}
+\`\`\`
+
```

Comment:
Should we ensure that we only show these lines if there are added dependencies?

Reply:
This is a great idea now that we can build this message dynamically!